tests/sentry/api/endpoints/test_organization_dashboard_widget_details.py

```diff
@@ -638,6 +638,8 @@ def test_save_with_invalid_orderby_not_from_columns_or_aggregates(self):
         assert "queries" in response.data, response.data
 
     def test_save_with_total_count(self):
+        # We cannot query the Discover entity without a project being defined for the org
+        self.create_project()
```

Comment:
If there's at least one project in the organization, we will use the list of projects matching `project_id`:
https://github.com/getsentry/sentry/blob/85558cb8f2b11fad6d74f048049ec33824624644/src/sentry/search/events/builder/discover.py#L134